docs/src/__examples__/Tooltip/DEFAULT.tsx

```diff
@@ -7,9 +7,35 @@ export default {
       <Button disabled>Book</Button>
     </Tooltip>
   ),
-  info: {
-    title: "Default tooltip",
-    description:
-      "By default, tooltips will open on hover over their child on large devices and a click on the child on smaller devices. By default, preference is given to aligning to the right and in the center of the child.",
-  },
+  exampleKnobs: [
+    {
+      component: "Tooltip",
+      knobs: [
+        { name: "stopPropagation", type: "boolean", defaultValue: false },
+        { name: "lockScrolling", type: "boolean", defaultValue: true },
+        { name: "renderInPortal", type: "boolean", defaultValue: false },
+        { name: "block", type: "boolean", defaultValue: false },
+        { name: "removeUnderlinedText", type: "boolean", defaultValue: false },
+        { name: "enabled", type: "boolean", defaultValue: true },
+        {
+          name: "preferredAlign",
+          type: "select",
+          options: ["center", "start", "end"],
+          defaultValue: "",
+        },
+        {
+          name: "preferredPosition",
+          type: "select",
+          options: ["right", "left", "top", "bottom"],
+          defaultValue: "",
+        },
+        {
+          name: "size",
+          type: "select",
+          options: ["small", "medium"],
+          defaultValue: "small",
+        },
+      ],
+    },
+  ],
```

Comment:
An additional knob to open 🙏  Also, I'd remove knobs like `stopPropagation` and `renderInPortal`.